src/diff_tests/diff_tests.cpp

```diff
@@ -38,7 +38,6 @@ std::string extractData(const std::string str, std::string type)
     if(type == "read_write_svfg")
     {
         marker = "****SVFG Statistics****";
-        search.push_back("AvgTopLvlPtsSize");
```

Comment:
Why this is removed?

Reply:
we removed it because it's not part of the svfg statistics. do we add it back?

Reply:
I think we can state the FSPTA's `AvgTopLvlPtsSize`

Reply:
The marker for `AvgTopLvlPtsSize` should be `****Flow-Sensitive Pointer Analysis Statistics****`rather than `****SVFG Statistics****";`

Reply:
can you change and quickly check whether `AvgTopLvlPtsSize ` for  ****Flow-Sensitive Pointer Analysis Statistics**** still consistent for write and read?

Reply:
yep we will change and check AvgTopLvlPtsSize for Flow-Sensitive Pointer Analysis Statistics

Reply:
we have changed it and tested locally